app/javascript/controllers/media_tag_controller.js

```diff
@@ -45,11 +45,14 @@ export default class extends Controller {
       player.on('loadedmetadata', () => {
         const event = new CustomEvent('media-loaded', { detail: player })
         window.dispatchEvent(event)
-      })
-      player.on('timeupdate', () => {
-        const timestamp = player.currentTime()
-        const event = new CustomEvent('time-update', { detail: timestamp })
-        window.dispatchEvent(event)
+
+        player.off('loadedmetadata')
+
+        player.on('timeupdate', () => {
+          const timestamp = player.currentTime()
+          const event = new CustomEvent('time-update', { detail: timestamp })
+          window.dispatchEvent(event)
+        })
```

Comment:
Delay listening for `timeupdate` events until after `loadedmetadata` fires, else `timeupdate` fires before any captions are present.

Reply:
Can you make that a code comment?

Reply:
@jcoyne Done.